### PR TITLE
Updated (all) configurations to VS2017

### PR DESCRIPTION
**Pull request by FireEmerald**
**Todo:** After merge, update also SoundSwitch to use this, otherwise you are unable to compile SoundSwitch with VS2017 without manual changes.

### CODE REVIEW

File: Audio.EndPoint.Controller.Wrapper/Audio.EndPoint.Controller.Wrapper.vcxproj [hunk 1]
```diff
@@ -31,27 +31,27 @@
     <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
     <Keyword>ManagedCProj</Keyword>
     <RootNamespace>AudioEndPointControllerWrapper</RootNamespace>
-    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.14393.0</WindowsTargetPlatformVersion>
```
Comment (Belphemur) [inline Audio.EndPoint.Controller.Wrapper/Audio.EndPoint.Controller.Wrapper.vcxproj]: The target need to stay windows 8.1 else it removes support for Windows 7 and 8. Windows 10 toolset is not backward compatible.

https://stackoverflow.com/questions/34706246/visual-studio-2015-change-target-platform-to-windows-7

Same for the other files. Only change the Platform toolset.

**Review by Belphemur — CHANGES_REQUESTED** [review]
Need to revert the change in WindowsTargetPlatformVersion

**Comment by FireEmerald** [comment]
Reverted, but the ToolsVersion should be 15.0 as shown here.
https://stackoverflow.com/questions/43078438/building-msbuild-15-project-programmatically

**Review by Belphemur — APPROVED** [review]

**Comment by Belphemur** [comment]
yes the tools version should be 15.0. For once, the tooling is backward compatible :)

Thanks for this PR.

**Pull request merged**